go-eicio/event.go

```diff
@@ -79,12 +52,16 @@ func (evt *Event) AddCollection(collection Message, name string) {
 	}
 	evt.Header.Collection = append(evt.Header.Collection, collHdr)
 	evt.payload = append(evt.payload, collBuf...)
+
+	return nil
 }
 
-func (evt *Event) GetCollection(name string) Message {
+var BlankColl = errors.New("collection not found or type is blank")
```

Comment:
FYI, exported error values are usually called `ErrSomething` in idiomatic Go:

```
$> go doc io.ErrClosedPipe 
var ErrClosedPipe = errors.New("io: read/write on closed pipe")
    ErrClosedPipe is the error used for read or write operations on a closed
    pipe.

```

and error types are usually called `SomethingError`:

```
$> go doc os.PathError
type PathError struct {
	Op   string
	Path string
	Err  error
}
    PathError records an error and the operation and file path that caused it.

```